routes/owners/api.js

```diff
@@ -89,7 +89,14 @@ export function index(req: Object, res: Object) {
     return body && body.pull_request && prActionTypes.indexOf(body.action) > -1;
   }
 
+  function isRetryCommand(body) {
+    return body && body.issue && body.comment && body.action == 'created' &&
+        body.issue.pull_request && body.comment.body.trim().toLowerCase() ==
```

Comment:
This can probably be a little less strict. Maybe just `.contains` with the username and retry, but no punctuation.